config/application.rb

```diff
@@ -1,6 +1,19 @@
 require File.expand_path('../boot', __FILE__)
 require File.expand_path('../preinitializer', __FILE__)
-require 'rails/all'
+require "rails"
+%w(
+  active_record
+  action_controller
+  action_view
+  action_mailer
+  active_job
+  sprockets
```

Comment:
Yeah I don't see another way to fetch them :/ @matthewd Do you have any suggestions?

I feel like Rails should allow you to skip loading rake tasks in railties. It seems like people making their own rake tasks specific to their test suites and ignoring gems' additions is a common thing. Maybe I should leave this change out for now and sometime pursue that instead, if anyone else agrees that it is of value.